coreos-4/policies/SA-System_and_Services_Acquisition/component.yaml

```diff
@@ -295,151 +336,225 @@
   implementation_status: planned
   narrative:
     - text: |
-        'A control response to SA-4 (9) is planned.'
+        A control response to SA-4(9) is planned. Progress
+        can be tracked at:
+
+        https://issues.redhat.com/browse/CMP-366
 
 - control_key: SA-4 (10)
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: not applicable
   narrative:
     - text: |
-        'This control reflects organizational procedure/policy and is not
-        applicable to component-level configuration.'
+        This control reflects organizational procedure/policy and is not
+        applicable to component-level configuration.
 
 - control_key: SA-5
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: not applicable
+  implementation_status: planned
   narrative:
-    - text: |
-        'This control reflects organizational procedure/policy and is not
-        applicable to component-level configuration.'
+    - key: a
+      text: |
+        A control response to SA-5(a) is planned. Progress
+        can be tracked at:
+
+        https://issues.redhat.com/browse/CMP-367
+    - key: a.1
+      text: |
+        A control response to SA-5(a)(1) is planned. Progress
+        can be tracked at:
+
+        https://issues.redhat.com/browse/CMP-367
+    - key: a.2
+      text: |
+        A control response to SA-5(a)(2) is planned. Progress
+        can be tracked at:
+
+        https://issues.redhat.com/browse/CMP-367
+    - key: a.3
+      text: |
+        A control response to SA-5(a)(3) is planned. Progress
+        can be tracked at:
```

Comment:
Looking at the FedRAMP templates, it looks like we only need to have a response at the granularity of SA-5(a b c).

Reply:
Reference: https://www.fedramp.gov/assets/resources/templates/FedRAMP-SSP-High-Baseline-Template.docx